WebHDFS.Kitty.IntegrationTests/Delete.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+
+namespace WebHDFS.Kitty.IntegrationTests
+{
+
+    public class Delete
+    {
+        private IWebHdfsClient client;
+
+        [CheckConnStrSetupFact]
+        public async Task Remove()
```

Comment:
1) We should prepare hdfs for test: add `sample2` file
2) Check file after remove